### PR TITLE
wishlist redesign

### PR DESCRIPTION
**Pull request by tarasSanotskyi**
## Description
Redesign wishlist page according to the new design on FE 
Cleare error in console
Create tests for Modal component

PRJPG

PR2

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/components/modal/modal.styles.js [hunk 1]
```diff
@@ -2,43 +2,85 @@ import { makeStyles } from '@material-ui/core/styles';
...
     position: 'absolute',
+    fontFamily: 'Open sans',
     top: '50%',
     left: '50%',
     transform: 'translate(-50%, -50%)',
-    width: 400,
-    backgroundColor: theme.palette.background.paper,
-    border: '2px solid #000',
+    width: 480,
+    // backgroundColor: theme.palette.background.paper,
```
Comment (Skrekliam) [inline src/components/modal/modal.styles.js:15]: Please remove comment

File: src/components/modal/modal.styles.js [hunk 2]
```diff
@@ -2,43 +2,85 @@ import { makeStyles } from '@material-ui/core/styles';
...
     marginTop: '10px',
+    padding: '0 7px 7px 0',
     '& > button': {
-      color: theme.palette.button.normal.color,
-      backgroundColor: theme.palette.button.normal.backgroundColor,
+      color: theme.palette.white,
+
+      backgroundColor: theme.palette.black,
+      fontSize: 14,
+      // fontColor: theme.palette.black,
```
Comment (Skrekliam) [inline src/components/modal/modal.styles.js:70]: Remove comment

**Review by Skrekliam — CHANGES_REQUESTED** [review]

File: src/components/modal/modal.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ import Button from '@material-ui/core/Button';
 import Popper from '@material-ui/core/Popper';
 import { useStyles } from './modal.styles';
 
+import closeIcon from '../../images/modal-icon/modalCloseIcon.svg';
```
Comment (maarekua) [inline src/components/modal/modal.js:9]: Use material icons.

**Review by maarekua — CHANGES_REQUESTED** [review]

**Comment by Skrekliam** [comment]
Please add screenshot of console. It should be clear.

File: src/components/modal/modal.styles.js (hunk 1 above)
Comment (tarasSanotskyi) [reply src/components/modal/modal.styles.js]: done

File: src/components/modal/modal.styles.js (hunk 2 above)
Comment (tarasSanotskyi) [reply src/components/modal/modal.styles.js]: done

File: src/components/modal/modal.js (hunk 3 above)
Comment (tarasSanotskyi) [reply src/components/modal/modal.js]: done

File: src/components/modal/tests/modal.spec.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      language: 1,
+      message: 'test',
+      isOpen: true,
+      onAction: jest.fn()
+    };
+    render(
+      <SnackBarContextProvider>
+        <Modal {...props} />
+      </SnackBarContextProvider>
+    );
```
Comment (Skrekliam) [inline src/components/modal/tests/modal.spec.js:90]: Why you dont use this in beforeEach?

Comment (tarasSanotskyi) [reply src/components/modal/tests/modal.spec.js]: fixed

**Review by bohdanst1914 — APPROVED** [review]

**Review by Skrekliam — APPROVED** [review]

**Pull request merged**